examples/src/main/res/values/strings.xml

```diff
@@ -44,6 +44,9 @@
     <string name="title_arrival_ui_building_extrusions_kotlin" translatable="false">Arrival UI Building Extrusions Kotlin</string>
     <string name="description_arrival_ui_building_extrusions_kotlin" translatable="false">Show building extrusions once the device is close enough to the final destination. Example\'s in Kotlin.</string>
 
+    <string name="title_instruction_view" translatable="false">Navigate with InstructionView</string>
```

Comment:
Let's change the title to 
`<string name="title_instruction_view" translatable="false">InstructionViewActivity</string>`